apps/chat/src/utils/app/data/prompt-service.ts

```diff
@@ -71,3 +71,30 @@ export const getPreparedPrompts = ({
         : path,
     };
   }); // to send prompts with proper parentPath
+
+const isRootPromptId = (id?: string) =>
+  isRootId(id) && id?.startsWith(`${ApiKeys.Prompts}/`);
+
+export const getImportPreparedPrompts = ({
+  prompts,
+  folders,
+}: {
+  prompts: Prompt[];
+  folders: FolderInterface[];
+}) =>
+  prompts.map((prompt) => {
+    const { path } = getPathToFolderById(folders, prompt.folderId, true);
+    const newName = prompt.name.replace(notAllowedSymbolsRegex, '');
```

Comment:
```suggestion 
    const newName = prepareEntityName(prompt.name);
```
name can't be more than 160 symbols now.

Reply:
Thank you for the suggestion!
New prepareEntityName function used in all needed places.